language-design.md

```diff
@@ -62,7 +129,37 @@ the commenters?
+
+ 
+They raise a good point - programmers handle a lot of really important, really sensitive things that should certainly be done as well as possible.  Furthermore, the purpose of the tool is to design it to be usable by the programmers once they have learned their craft - something designed for novices may end up being used exclusively by novices. 
+
+However, perhaps the most important point that programmers make is that there is much, much more to software development than just the lines of code.
+
+> The key to understandable code (apart from being well structured..and formed..) is appropriate commenting.
+
+> Oh and hopefully some unit tests, or TDD, and some autodoc....and maybe reSharper...oh and using common design patterns...oh and.
+
+> Wait a second, this programming thing isn't just about the actual code! [Comment by "Drew Brigham" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-1da48980-66bd-11e1-921b-9794fc737ae8))] [Pavlus, 2012].
+
+While an easier language to learn may help to some extent, there are many other  important (and often harder) concepts that are necessary. 
+
+On the other hand, it is maybe irresponsible to dismiss the complaints of those who are less technically fluent. We don't want to make the mistake of conflating the problem with the flawed suggested solution, thereby discarding both.
+
+The `repeat` code example given in the article is an excellent microcosm of this point. Several commenters correctly point out that `repeat` is actually a horrible solution. You almost never want to loop some fixed number of times, that's just incidental. The actual concept being expressed is that you want to do something for each element of some collection. The standard for-loop notation is much more flexible and (eventually) understandable than the `repeat` notation. However, although `repeat` is a bad idea, revisiting for-loops is not. If Guido van Rossum had not paid attention to this kind of complaint, we would never have gotten Python.
+
+A lot of advances we've made past older languages that are now ridiculed for being obtuse came from listening. We should accept that these are real problems, with non-obvious and tradeoff-y solutions, but that solving them would be a *good* thing. The real point that should be made to people who bring up complaints like these isn't that they're foolish and don't know what they're talking about. It's that despite first appearances, these issues are extremely difficult to solve well, but we are indeed going to try.
```

Comment:
It's that beautiful moment when we succeed at solving these issues that a new, more fluent language is born. :-)